resources/js/processes/screen-builder/screen.vue

```diff
@@ -615,6 +615,7 @@ export default {
         this.$refs.menuScreen.sectionRight = false;
       }
       this.mode = mode;
+      this.$store.commit("globalErrorsModule/setMode", this.mode);
```

Comment:
Consider to use map mutation.
see: https://blog.logrocket.com/best-practices-mapping-vuex/#vuexmapmutations